ExtractorUtils/config/remote_config.yml

```diff
@@ -0,0 +1,39 @@
+# Sample configuration file for remote config. This can be used as remote config directly.
```

Comment:
Could this file have been included by mistake? Doesn't seem to be used.

Reply:
It's a reference file, meant to be what this PR makes possible.